std_msgs/Header header

geometry_msgs/Point end_pos
geometry_msgs/Quaternion end_quat
geometry_msgs/Point cam_pos
geometry_msgs/Quaternion cam_quat

geometry_msgs/Wrench wrench

 